texthero/_types.py

```diff
@@ -81,68 +82,70 @@ class HeroSeries(pd.Series):
     3. VectorSeries: Every cell is a vector representing text, i.e.
     a list of floats. For example, `pd.Series([[1.0, 2.0], [3.0]])` is a valid VectorSeries.
 
-    4. RepresentationSeries: Series is multiindexed with level one
-    being the document, level two being the individual features and their values.
+    4. DataFrame: A Pandas DataFrame representing a matrix.
```

Comment:
The same discussion as before; just mention we sub-class also from DataFrame to allow this type as input for specific functions

Reply:
will change